Mathlib/MeasureTheory/Group/Action.lean

```diff
@@ -103,6 +104,28 @@ theorem map_smul : map (c • ·) μ = μ :=
 
 end MeasurableSMul
 
+section SMulHomClass
+
+variable {M : Type u} {α : Type v} {β : Type w}
+  [MeasurableSpace M] [MeasurableSpace α] [MeasurableSpace β]
+
+theorem SMulInvariantMeasure_map {F : Type _} [SMul M α] [SMul M β] [SMulHomClass F M α β]
+    [MeasurableSMul M β]
+    (μ : Measure α) [SMulInvariantMeasure M α μ] (f : F) (hf: Measurable f) :
+    SMulInvariantMeasure M β (map f μ) := by
+  refine' ⟨fun m S hS => _⟩
+  calc
+    map f μ ((m • ·) ⁻¹' S) = μ (f ⁻¹' ((m • ·) ⁻¹' S)) := map_apply hf (MeasurableSet.preimage
+      hS (measurable_const_smul _))
+    _ = μ (((m • ·) ∘ (f : α →[M] β)) ⁻¹' S) := by rw [@preimage_comp]; rfl
```

Comment:
@eric-wieser Hey, If you would look at the PR one last time it would be great!
In particular, if you have an idea as to how to avoid the annoying ; rfl here it would be much appreciated!

Reply:
I attempted a fix, but will need to wait for CI again to look further.

Reply:
Looks like it passed, thanks!